src/sc/sidechainTxsCommitmentBuilder.cpp

```diff
@@ -161,6 +161,14 @@ bool SidechainTxsCommitmentBuilder::add_fwt(const CTxForwardTransferOut& ccout,
          &ret_code
     );
 
+    if (ret) {
+    LogPrint("sc", "%s():%d - scTxsCommitment builder: successfully added FT for sidechain scId[%s].\n",
+            __func__, __LINE__, ccout.GetScId().ToString());
+    } else {
+        LogPrint("sc", "%s():%d - scTxsCommitment building failed when adding FT for sidechain scId[%s] (CCTPlib error).\n",
+            __func__, __LINE__, ccout.GetScId().ToString());
+    }
+
```

Comment:
We saw in the past some performance issues related to the printing of debug logs when dealing with the commitment tree. This is mainly because additions to the tree can happen quite frequently.

What do you think about printing the line only in case of error and doing nothing if the operation succeeds?

Reply:
Agree. Removed from both `SidechainTxsCommitmentBuilder` and `SidechainTxsCommitmentGuard`